src/project/ProjectArea.tsx

```diff
@@ -24,68 +23,68 @@ const ProjectArea = ({
   selectedFile,
   onSelectedFileChanged,
 }: ProjectAreaProps) => {
-  const { files, name: projectName } = useProject();
+  let { files, name: projectName } = useProject();
+  //files = Array(20).fill(files[0]);
```

Comment:
Should revert once no longer needed for testing scrollable file list area.

Reply:
Done in b46219c64b4c0cf34766bbc04bf537f42c38e434